README.md

```diff
@@ -1,13 +1,18 @@
 # OpenVPN for Docker
 
-**the original project - [jpetazzo/dockvpn](https://github.com/jpetazzo/dockvpn)** and it has its own [automatic build on dockerhub](https://hub.docker.com/r/jpetazzo/dockvpn/). 
+**the original project - [jpetazzo/dockvpn](https://github.com/jpetazzo/dockvpn)** and it has its own [automatic build on dockerhub](https://hub.docker.com/r/jpetazzo/dockvpn/).
+
 
- 
 Quick instructions:
 
 ```bash
-CID=$(docker run -d --privileged -p 1194:1194/udp -p 443:443/tcp umputun/dockvpn)
-docker run -t -i -p 8080:8080 --volumes-from $CID umputun/dockvpn serveconfig
+CID=$(docker run -d --privileged -p 1194:1194/udp -p 443:443/tcp -e "NO_LOGS_PLEASE=1" --log-driver none umputun/dockvpn)
+docker run -t -i -p 8080:8080 -e "NO_LOGS_PLEASE=1" --log-driver none --volumes-from $CID umputun/dockvpn serveconfig
```

Comment:
pls don't change the command as the default, but rather add an example "running without logs"

Reply:
Fixed